### PR TITLE
style(footer): correct spacing

### PR DESCRIPTION
**Pull request by dfvalero**

### CODE REVIEW

File: src/components/molecules/ZopaFooter/ZopaFooter.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const ZopaFooter = ({ baseUrl = 'https://www.zopa.com', ...rest }: IFooterProps)
   <Footer {...rest}>
```
Comment (moshie) [inline src/components/molecules/ZopaFooter/ZopaFooter.tsx:20]: You could achieve the padding on the padding on the footer like this:
```
const ZopaFooter = ({ baseUrl = 'https://www.zopa.com', className = 'pt-10 l:pt-11 mb-9' ...rest }: IFooterProps) => (
  <Footer className={className} {...rest}>
```

That would remove the need for the `Footer` component

Comment (dfvalero) [reply src/components/molecules/ZopaFooter/ZopaFooter.tsx:20]: nope! we can't do that. If we set a class to `<ZopaFooter />` we will overwrite the internal class.

Comment (moshie) [reply src/components/molecules/ZopaFooter/ZopaFooter.tsx:20]: Fair enough

**Pull request merged**